games-misc/aceofwands/aceofwands-1.0.4.recipe

```diff
@@ -0,0 +1,47 @@
+SUMMARY="A tarot card reading application"
+DESCRIPTION="Ace of Wands is a desktop application for Haiku that provides tarot card readings."
+HOMEPAGE="https://github.com/1wErt3r/Ace_of_Wands"
+COPYRIGHT="2025 politebot"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/1wErt3r/Ace-of-Wands/archive/refs/tags/v$portVersion.tar.gz"
+SOURCE_DIR="Ace-of-Wands-$portVersion"
+CHECKSUM_SHA256="0a018318f5cea1970598d284c1303d458450536e62e5e84c71ac6818ee11b487"
+
+ARCHITECTURES="all"
```

Comment:
```suggestion
ARCHITECTURES="all !x86_gcc2"
```

gcc2 can be disabled. It will not work anyway because the dependencies are not available for gcc2.

Reply:
It looks like you accidentally reverted some changes.